spec/decorators/article_decorator_spec.rb

```diff
@@ -6,10 +6,15 @@ def create_article(*args)
     article.decorate
   end
 
+  subject(:decorated_article) { build(:article).decorate }
+
   let(:article) { build(:article) }
   let(:published_article) { create_article(published: true) }
   let(:organization) { build(:organization) }
 
+  it { is_expected.to respond_to(:can_be_featured_in_feed) }
```

Comment:
Nitpicky question: do we care about whether these are actually aliases or not? The answer is probably no but the two separate `responds_to` checks theoretically allow for drift, which something like this wouldn't:

```ruby
decorated_article = build(:article).decorate
expect(decorated_article.method(:can_be_featured_in_feed))
  .to eq(decorated_article.method(:can_be_featured_in_feed?))
```